app/views/users/show.html.erb

```diff
@@ -0,0 +1,5 @@
+</p><h1><%=@user.name%></h1>
+</p><h5><%=@user.university.name%></h5>
+
+</br>
```

Comment:
Erm. WHy do you need it.
Also used wrongly, supposed to be `<br />`. Also it's a hack, you shouldn't need to use it most of the time